### ISSUE TITLE
Explainer is out of date

### ISSUE THREAD

**Issue by AdamTReineke** [slightlyoff/IntersectionObserver#49]
Edit: Overall, the Explainer is out of date. Needs to be updated to the current spec. Below describes just one difference, not comprehensive.

---------------

The interfaces don't have the same methods. The spec has `DOMString threshold = "1px"`, the explainer has `boolean thresholdCallbacks = true`.

https://github.com/slightlyoff/IntersectionObserver/blob/master/index.bs#L235
```
dictionary IntersectionObserverInit {
  ..[snip]..
  DOMString threshold = "1px";
};
```

https://github.com/slightlyoff/IntersectionObserver/blame/master/explainer.md#L54

(Sorry for linking to the blame, I couldn't figure out how to link directly to a line in the markdown)
```
dictionary IntersectionObserverInit {
  ..[snip]..
  // Whether to give callbacks only when an element starts/stops intersecting
  // the root bounds or everytime it changes how much it intersects.
  // Callback only fires if the element isn’t intersecting an edge of the
  // viewport in the case that the element jumps from being entirely outside
  // the viewport to entirely inside it.
  // Defaults to true, a less power-hungry option.
  boolean thresholdCallbacks = true;
};
```

**Comment by mpb** [comment]
Explainer is out of date, by several iterations. 
We should update (or remove it?)

**Comment by ojanvafai** [comment]
We should update it. Explainers are good for web developers trying to learn
how to use the API and whether it works for them.

**Comment by ojanvafai** [comment]
Fixed in 1bb9e37
